.kokoro/client-library-check.sh

```diff
@@ -60,14 +119,12 @@ fi
 
 pushd ${REPO}
 
-# replace version
-xmllint --shell <(cat pom.xml) << EOF
-setns x=http://maven.apache.org/POM/4.0.0
-cd .//x:artifactId[text()="google-cloud-shared-config"]
-cd ../x:version
-set ${VERSION}
-save pom.xml
-EOF
+# Replace sdk-plaform-java-config version in java-spanner and java-pubsub.
+if [ "$REPO" == "java-spanner" ] || [ "$REPO" == "java-pubsub" ]; then
```

Comment:
will this get removed / updated when you update all the other handwritten libraries for your change?

Reply:
Yup, that's right. I've modified this comment to a TODO and created a follow-up issue to update the script: https://github.com/googleapis/java-shared-config/issues/748